### PR TITLE
DOC/WIP: travis notebook doc build

### PR DESCRIPTION
**Pull request by TomAugspurger**
Just want to see the output log from the pull request.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
https://travis-ci.org/pydata/pandas/jobs/139080123

File: ci/requirements-2.7_DOC_BUILD.run [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ipython
+ipykernel
```
Comment (jreback) [inline ci/requirements-2.7_DOC_BUILD.run]: I think you can just include ``notebook`` and get all of the deps

**Comment by TomAugspurger** [comment]
@jorisvandenbossche so my last commit succeeded https://travis-ci.org/pydata/pandas/jobs/139093566#L1569

I'm not sure why the `python -m ipykernel install` was necessary, I've never had to do that on my own system. I want to look into the correct way to do this, but won't have a chance till tonight or tomorrow.

**Comment by jorisvandenbossche** [comment]
Hmm, I am not sure this is due to the installing of the kernel, as the logs say:

```
python -m ipykernel install
[Errno 13] Permission denied: '/usr/local/share/jupyter'
```

So I think it is just adding the `ipykernel` as dependency which was needed?

**Comment by TomAugspurger** [comment]
Ahh, I missed that. Let me retry just using `notebook` to get everything like Jeff suggested.

**Comment by jorisvandenbossche** [comment]
In principle, notebook is not needed (nbconvert and ipykernel should suffice as well), although doesn't hurt of course :-)

File: ci/build_docs.sh [hunk 2]
```diff
@@ -19,6 +19,8 @@ fi
 
 if [ x"$DOC_BUILD" != x"" ]; then
 
+    jupyter-kernelspec list
```
Comment (jorisvandenbossche) [inline ci/build_docs.sh]: This can be removed again I think?

**Comment by jorisvandenbossche** [comment]
Apart from the clean-up, I think this can be merged?

**Comment by TomAugspurger** [comment]
@jorisvandenbossche fixed

**Comment by TomAugspurger** [comment]
@jorisvandenbossche http://pandas-docs.github.io/pandas-docs-travis/style.html 🎉 

A few minor things, like dependencies on ipywidgets and seaborn didn't work. I can do another followup.

**Comment by jorisvandenbossche** [comment]
Yes, we're coming closer :-)

Also a unicode thing (just needs a `u""` I think)

**Pull request merged**